scripts/ci/images/ci_build_dockerhub.sh

```diff
@@ -113,7 +113,7 @@ else
     export DOCKER_CACHE="local"
     # Name the image based on the TAG rather than based on the branch name
     export FORCE_AIRFLOW_PROD_BASE_TAG="${DOCKER_TAG}"
-    export INSTALL_AIRFLOW_VERSION="${DOCKER_TAG%-python*}"
+    export INSTALL_AIRFLOW_VERSION="==${DOCKER_TAG%-python*}"
```

Comment:
The line below now fails

Reply:
```
export AIRFLOW_CONSTRAINTS_REFERENCE="constraints-${INSTALL_AIRFLOW_VERSION}"
```

Reply:
Also the one in https://github.com/apache/airflow/blob/master/docs/apache-airflow/production-deployment.rst is `AIRFLOW_INSTALL_VERSION` and the one here is `INSTALL_AIRFLOW_VERSION`

Reply:
Yep. Good catch. Fixed.